lib/features/label-editing/cmd/UpdateLabelHandler.js

```diff
@@ -104,11 +104,6 @@ export default function UpdateLabelHandler(modeling, textRenderer) {
 
     var text = getLabel(label);
 
-    // don't resize without text
```

Comment:
Do you know why we added this in the first place? Is it due to some label related legacy?